Samples/iOS-Swift/iOS-Swift/AppDelegate.swift

```diff
@@ -19,7 +19,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             }
             options.debug = true
             // Sampling 100% - In Production you probably want to adjust this
-            options.tracesSampleRate = 1.0
+            options.tracesSampleRate = 0.001
```

Comment:
```suggestion
            options.tracesSampleRate = 1.0
```

for testing 0.001 makes sense but for debugging its always easier with 1.0